examples/benchmark/tls_bench.c

```diff
@@ -1558,6 +1636,54 @@ static void ShowCiphers(void)
         fprintf(stderr, "%s\n", ciphers);
 }
 
+static int SetupSupportedGroups(int verbose) {
```

Comment:
Start of function brace on its own line.

Reply:
This function only applies / is supported if TLS v1.3 is enabled. `WOLFSSL_TLS13`

Reply:
> Start of function brace on its own line.
> This function only applies / is supported if TLS v1.3 is enabled. WOLFSSL_TLS13

Will fix